runner/runner.go

```diff
@@ -995,6 +995,16 @@ retry:
 	if scanopts.Unsafe {
 		req.Header.Add("Connection", "close")
 	}
+	var faviconResp *httpx.Response
```

Comment:
favicon request should be moved after the [primary probe](https://github.com/projectdiscovery/httpx/blob/78762865305f08a6b2788e8a2040a80a71921139/runner/runner.go#L998). For example it can be moved at: https://github.com/projectdiscovery/httpx/blob/78762865305f08a6b2788e8a2040a80a71921139/runner/runner.go#L1348

Reply:
Done